Dockerfile.windows

```
@@ -0,0 +1,108 @@
+
+# Stage 3: Prebuilt - base + built Bun binary
+FROM base AS prebuilt
+
+# Set build environment variables for Windows
+ENV ENABLE_ZIG_ASAN=0
+ENV BUN_DEBUG_QUIET_LOGS=1
+ENV BUN_GARBAGE_COLLECTOR_LEVEL=0
+ENV BUN_FEATURE_FLAG_INTERNAL_FOR_TESTING=1
+
+# Build Bun
+RUN $env:PATH = $env:PATH + ';' + [System.Environment]::GetEnvironmentVariable('PATH','Machine') + ';C:\Users\ContainerAdministrator\.bun\bin'; `
```

Hadolint (2.14.0)

[error] Parsing stopped here. Invalid use of parentheses?

(SC1088)